Fedora/README.md

```diff
@@ -0,0 +1,13 @@
+##Its Strengthes :)
+--- 
+* Easy to use, simple, and beginner friendly 
+* 100% free 
+* Less hardware-consuming as it consumes ONLY 1GB of RAM
```

Comment:
Hmm, the point about hardware consumption could be tricky to prove. The [docs](https://docs.fedoraproject.org/en-US/fedora/latest/release-notes/welcome/Hardware_Overview/#hardware_overview-specs) recommend a minimum of 2GB for the default installation.

How about instead of this point we add:
- Good balance between bleeding edge and stability